text/103-leaving-x-twitter.md

```diff
@@ -0,0 +1,129 @@
+Drivers: Théo Zimmermann (@Zimmi48)
+---
+
+## Summary
+
+This RFC proposes to reform the management of the official social media channels of Coq in the context of the renaming to the Rocq Prover. In particular, it proposes to: search for additional volunteers to help manage them, and to revisit which social media platforms to be present on in the current context.
+
+## Context and motivation
+
+During the last four years, the Coq project's social media presence has been limited to the CoqLang account on Twitter (then X). This account was initially managed by three persons: Emilio Jesús Gallego Arias, Anton Trunov, and Théo Zimmermann. However, in the last two years, the account has been mostly managed by Emilio Jesús Gallego Arias alone (Anton Trunov is not active anymore in the Coq ecosystem, and Théo Zimmermann has stopped using X/Twitter).
+
+Leaving the management of the account on the shoulders of a single person is not sustainable in the long term. If the project wants to maintain a social media presence, it needs to involve more people in the management of its social media account(s).
+
+The CoqLang account uses the Coq-community logo instead of the Coq logo, because it is largely considered as a better visual identity for the Coq project. However, the CoqLang account is still the official account of the Coq project. In the context of the renaming of Coq to the Rocq Prover, we have to decide what to do with respect to our social media presence. If we keep the CoqLang account, we will have to rename it and to update its visual identity to use the new Rocq Prover logo.
+
+With the change of ownership of X/Twitter and the associated changes in moderation policies, the general social media context is much different than it was four years ago when it was decided to create the CoqLang account. In particular, many academics have left X/Twitter and moved to alternative platforms such as Mastodon (or more recently Bluesky). More recently, many journalists have also started to leave X/Twitter. Beyond individual decisions, there are now media (such as [The Guardian][guardian] in the UK, [La Vanguardia][lavanguardia] in Spain, [NPR][npr] in the US, [Ouest France][ouestfrance] and [Mediapart][mediapart] in France) and academic institutions (e.g., [Aix-Marseille University][amu], [University of Strasbourg][unistra], [University of Luxembourg][unilux], [RWTH Aachen][rwth], [Utrecht University][uu], [York University][york]) that are officially leaving X/Twitter as well.
+
+[guardian]: https://www.theguardian.com/media/2024/nov/13/why-the-guardian-is-no-longer-posting-on-x
+[lavanguardia]: https://www.lavanguardia.com/vida/20241114/10105193/vanguardia-dejara-publicar-x-convertido-red-desinformacion.html
+[npr]: https://www.npr.org/2023/04/12/1169269161/npr-leaves-twitter-government-funded-media-label
+[ouestfrance]: https://www.lemonde.fr/pixels/article/2024/11/19/ouest-france-annonce-suspendre-ses-publications-sur-le-reseau-social-x_6403278_4408996.html
```

Comment:
This is not political?

> According to the press release from France's leading regional press, this decision was taken due to the recent appointment of Elon Musk to the Trump administration, but also due to the change in the granting of "blue stickers".

Musk will be advisor but is not expected to hold any government position.